### PR TITLE
[Refactor] Search CSS

### PR DESCRIPTION
**Pull request by imjuangarcia**
Had to create a specific `searchModal.css` file, and import it globally, so we can style the classes Algolia uses internally.

### CODE REVIEW

File: components/utilities/search.js [hunk 1]
```diff
@@ -150,20 +152,26 @@ const Search = () => {
 
     return (
       <article
-        className={`item ${
-          props.hit.__position === indexFocus ? "focused" : ""
-        }`}
+        className={`
+          ${styles.HitContainer}
+          ${props.hit.__position === indexFocus ? styles.ActiveHit : ""}
+        `}
```
Comment (tvst) [inline components/utilities/search.js:158]: Use `classNames()` utility.

Same below.

File: components/utilities/search.js [hunk 2]
```diff
@@ -196,15 +204,23 @@ const Search = () => {
...
               // left: '-40em'
             }}
-            className="algolia"
+            className={styles.ModalContainer}
           >
             <FocusTrap>
               <div
-                className="modalContainer"
+                className={styles.Modal}
+                style={{ background: "rgb(163, 168, 184, .6)" }}
```
Comment (tvst) [inline components/utilities/search.js:212]: Why use inline styles? Can this be put in a class instead? Same below.

Also: why not use tailwind? For example: `bg-gray-40/60`

**Comment by tvst** [comment]
This breaks some of the existing design.

### Text input background color

Today:
image

This PR:
image

---

### Underline and result category styling

Today:
image

This PR:
image

---

**Review by tvst — CHANGES_REQUESTED** [review]

File: components/utilities/search.js (hunk 2 above)
Comment (imjuangarcia) [reply components/utilities/search.js]: Fixed on f0b5331

**Comment by imjuangarcia** [comment]
> This breaks some of the existing design

Great catch, thanks! Both were fixed on 57e8eef and 226ac21

File: components/utilities/search.js [hunk 3]
```diff
@@ -150,20 +153,29 @@ const Search = () => {
...
-          <section className="image_container bg-yellow-90">
-            <div className={`icon-${icon}`}>
-              <i>{icon}</i>
+        <a
+          className={classNames("not-link", styles.HitLink)}
+          href={props.hit.url}
+        >
+          <section className={styles.IconContainer}>
+            <div
+              className={`
```
Comment (tvst) [inline components/utilities/search.js:168]: nit: use `classNames`

File: components/utilities/search.js [hunk 4]
```diff
@@ -231,11 +250,20 @@ const Search = () => {
   }
 
   let searchBar = (
-    <section className="block-search">
-      <section className="hot-key small" onClick={searchClicked}>
-        <i>search</i>
-        <p className="search-text">Search</p>
-        {windowWidth > 1024 && <p className="command">{hotkey}</p>}
+    <section
+      className={`
```
Comment (tvst) [inline components/utilities/search.js:254]: `classNames`

**Review by tvst — APPROVED** [review]

File: components/utilities/search.js (hunk 4 above)
Comment (imjuangarcia) [reply components/utilities/search.js]: Fixed on bd22ce4

File: components/utilities/search.js (hunk 3 above)
Comment (imjuangarcia) [reply components/utilities/search.js]: Fixed on bd22ce4

**Pull request merged**